src/main/java/com/agilethought/atsceapi/exceptions/BadRequestException.java

```diff
@@ -0,0 +1,12 @@
+package com.agilethought.atsceapi.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.BAD_REQUEST)
+public class BadRequestException extends Exception{
+    public BadRequestException(String message) {
+        super(message);
+    }
+
+}
```

Comment:
I don't think creating exception classes and methods is part of your user story. Feel free to backup this code for later use, but for now it should not be in here.

Reply:
I think we can left this classes for future implementation, but this exception should be unauthorized for both cases

Reply:
So you're saying to leave the classes themselves but remove the implementation for now?

Reply:
As part of her Us is throw 401 exception, she needs to change this classes for one that throws 401 instead of 404 or 400 codes.

Reply:
Ah, I see got it.